CHANGELOG

```diff
@@ -65,6 +65,9 @@ Version 4.5.2, released on March 23, 2023
 - The following integrations have updated minimum version requirements:
   * RAJA >= 2022.10.3
 
+- Adds mfem::forall functions that can be used inplace of MFEM_FORALL macros,
+  which have been kept for backward compatibility.
+
```

Comment:
Move to the section for v4.5.3

Also, it is good to clarify that the new function allows for better debugging of lambda bodies compared to the `MFEM_FORALL` macros.